packages.json

```diff
@@ -37,6 +37,7 @@
 				"oddjob-mkhomedir",
 				"opendyslexic-fonts",
 				"playerctl",
+    "podman-machine",
```

Comment:
Correct location, but spacing doesn't match.

If you can fix that up, we'll be good to merge.